linux_os/guide/services/cron_and_at/package_cron_installed/rule.yml

```diff
@@ -42,5 +42,6 @@ template:
     name: package_installed
     vars:
         pkgname: cron
+        pkgname@rhel10: cronie
```

Comment:
Should a `pkgname@rhel10` key be added also in the template vars in rule `service_cron_enabled`?

Reply:
The `service_cron_enabled` rule seems to be wrong on all RHELs. The service is `crond`, not `cron`. I've checked rhel8, rhel9, and rhel10 and everywhere it's like that
```
# systemctl status cron
Unit cron.service could not be found.
# systemctl status crond
● crond.service - Command Scheduler
     Loaded: loaded (/usr/lib/systemd/system/crond.service; enabled; preset: enabled)
     Active: active (running) since Fri 2024-10-04 12:40:38 EDT; 33min ago
 Invocation: fd4397b628444aeb944ba7856f93f526
   Main PID: 1115 (crond)
      Tasks: 1 (limit: 11103)
     Memory: 1.1M (peak: 2.9M)
        CPU: 27ms
     CGroup: /system.slice/crond.service
             └─1115 /usr/sbin/crond -n
```
And I see the rule only in `default.profile` profiles, so it not used in any relevant rhel profile.

(but it seems to be relevant for other products, according to profiles)

Reply:
Checked RHEL distros and the service is named `crond`, not `cron`. Added new commit with the fix for RHEL distros